### PR TITLE
[DEV APPROVED] 10511 whatsapp contact panel

### PR DESCRIPTION
**Pull request by andreferreiradlw**
[TP10511](https://moneyadviceservice.tpondemand.com/restui/board.aspx?#page=board/5253959470622112123&appConfig=eyJhY2lkIjoiNTM0ODcyOThDQjBDMTMyMDgyMTBENEZEM0QwM0ZBQzkifQ==&boardPopup=userstory/10511/silent)

Style adjustments for the whatsapp footer contact panel.

[PR2132 in Frontend](https://github.com/moneyadviceservice/frontend/pull/2132)

### CODE REVIEW

File: bower_components/yeast [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/andre/.local/share/bower/links/yeast
```
Comment (margOnline) [inline bower_components/yeast]: Do we want dev specific references in the codebase?

File: assets/layout/common/_contact_panels.scss [hunk 2]
```diff
@@ -4,18 +4,10 @@
...
-      content: "";
-      position: absolute;
-      top: 0;
-      bottom: $baseline-unit*4;
-      left: 50%;
-      border-right: 1px solid $color-newsletter-input-border;
-      width: 1px;
-      min-height: 400px;
-    }
+    flex-direction: row;
```
Comment (margOnline) [inline assets/layout/common/_contact_panels.scss]: Nice use of flex options

File: assets/layout/common/_contact_panels.scss [hunk 3]
```diff
@@ -4,18 +4,10 @@
...
   @include respond-to($mq-l) {
-    &:after {
-      content: "";
-      position: absolute;
-      top: 0;
-      bottom: $baseline-unit*4;
-      left: 50%;
-      border-right: 1px solid $color-newsletter-input-border;
-      width: 1px;
-      min-height: 400px;
```
Comment (davidtrussler) [inline assets/layout/common/_contact_panels.scss]: I think this - or something like it - should be kept: I'm guessing we just need to set the value for `left` to 33.3% to restore the vertical rules between columns.

Comment (andreferreiradlw) [reply assets/layout/common/_contact_panels.scss]: The borders have been adjusted now

File: bower_components/yeast (hunk 1 above)
Comment (andreferreiradlw) [reply bower_components/yeast]: Fixed!

**Pull request merged**